web/packages/shared/components/TextEditor/TextEditor.jsx

```diff
@@ -17,11 +17,18 @@
  */
 
 import React from 'react';
+import styled from 'styled-components';
 import ace from 'ace-builds/src-min-noconflict/ace';
 
 import 'ace-builds/src-noconflict/mode-json';
 import 'ace-builds/src-noconflict/mode-yaml';
 import 'ace-builds/src-noconflict/ext-searchbox';
+import { ButtonSecondary } from 'design/Button';
+import Flex from 'design/Flex';
+import { Copy, Download } from 'design/Icon';
+import { copyToClipboard } from 'design/utils/copyToClipboard';
+import { downloadObject } from 'e-teleport/services/downloads';
```

Comment:
We can't import from e, `downloadObject` should be moved to OSS.

Reply:
TY, totally missed that. I've moved it to our `utils` package.